File: socketServer/receiver.js
var io = require('socket.io-client');

var open_socket = function () {
    var socket = io.connect('http://streaming.plenar.io/',{transports:['websocket','polling']});

    socket.on('data', function (data) {
        console.log(data);
    });
    socket.on('internal_error', function (err) {
        console.log(err);
    });
};

module.exports.open_socket = open_socket;

File: socketServer/receiver.py
import json
from socketIO_client import SocketIO
import time
from threading import Thread

def on_data(data):
    print json.dumps(data)


def on_error(err):
    print json.dumps(err)


def open_client():
    socketIO = SocketIO("streaming.plenar.io")

    socketIO.on('data', on_data)
    socketIO.on('internal_error', on_error)
    socketIO.wait()

for i in range(0, 5):
    t = Thread(target=open_client)
    t.start()
    time.sleep(.2)
